features/dd-sdk-android-flags/src/main/kotlin/com/datadog/android/flags/featureflags/internal/model/FlagsContext.kt

```diff
@@ -6,4 +6,13 @@
 
 package com.datadog.android.flags.featureflags.internal.model
 
+/**
+ * Internal context containing Datadog configuration needed for feature flag requests.
+ *
+ * @param applicationId The Datadog application ID. May be null when the SDK is not fully initialized
+ *                      or when running in certain test environments where app ID is not required.
+ * @param clientToken The client token for authenticating requests to Datadog
+ * @param site The Datadog site (e.g., US1, EU1) for routing requests
```

Comment:
is it possible to use `DatadogSite` enum here which we have https://github.com/DataDog/dd-sdk-android/blob/develop/dd-sdk-android-core/src/main/kotlin/com/datadog/android/DatadogSite.kt?

Reply:
Yes.. good idea.